src/client/helpers/index.ts

```diff
@@ -17,22 +17,22 @@ export const getNoteTitle = (text: string): string => {
   return noteText ? noteText[0].split(/\r?\n/)[0] : 'New note'
 }
 
-export const noteWithFrontmatter = (note: NoteItem): string =>
+export const noteWithFrontmatter = (note: NoteItem, category?: CategoryItem): string =>
   `---
 title: ${getNoteTitle(note.text)}
 created: ${note.created}
 lastUpdated: ${note.lastUpdated}
-category: ${note.category ? note.category : ''}
+category: ${category?.name ?? ''}
```

Comment:
Great use of the null coalescing operator! This is the first time I'm seeing it in use in a TS app.